src/filter_noise_other_project.py

```diff
@@ -0,0 +1,20 @@
+from src.lib import MessageFilters
+
+
+class FilterNOP(object):
+    @staticmethod
+    def filter_noise_other_project(messages):
+        """
+        Filters a list of  RQA and followup survey messages which have been labelled as Noise_Other_Project(NOP).
```

Comment:
Could probably be slightly clearer here in explaining that this is filtering out messages which have that label

Reply:
Updated thanks